create_pdfs.py

```diff
@@ -119,8 +109,7 @@ def main():
             )
             title = None
             if args.assign_title:
```

Comment:
There's a slight inconsistency here between `falsey` values.

I'd prefer to just explain this one in person instead of typing it all out,
so grab me when you have a minute.